crawl.py

```diff
@@ -31,12 +31,13 @@ def getURL(page):
 			continue
 		if item.find("..") != -1:
 			continue
-		if item.find("xjtu") != -1:
-			if item.find("http://") != -1:
-				tempList.append(item)
-			else:
-				tempUrl = baseUrl + str(item)
-				tempList.append(tempUrl)
+		if item.find("xjtu") == -1:
+			continue
+		if item.find("http://") != -1:
+			tempList.append(item)
+		else:
+			tempUrl = baseUrl + str(item)
+			tempList.append(tempUrl)
```

Comment:
Use Less if to make code more readable and pretty,
if XX or YY or ZZ ：
    continue
else:
  ...